index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="./images/google.png" type="image/x-icon">
+    <title>Google</title>
+</head>
+<body>
+
+    <div class="userinfo">
+        <ul>
+            <li> <a href="#">Gmail</a></li>
+            <li> <a href="#">Images</a></li>
+            <li> 
+                <div class="tooltip">
+                    <img src="./images/appGoogle.png" alt="google app" class="googleapp"><span class="toolTipText"> Google apps</span>
+                </div>
+                </li>
+            <li> 
+                <div class="tooltipletter">
+                    <span class="letterborder">K</span> <span class="tooltipemail"><span class="ga"> Google Account</span> <br/> Kanu Mike Chibundu <br/> Michotall95@gmail.com</span>
+                </div>
+            </li>
+        </ul>
+    </div>
+
+    <div class="headGoogle">
+        <img src="./images/googleimg.png" alt="google image" class="googleimg">
+        <!-- <h1><span class="gcolor">G</span><span class="firsto">o</span><span class="secondo">o</span><span class="gcolor">g</span><span class="lcolor">l</span><span class="firsto">e</span></h1> -->
```

Comment:
Delete unused code,  so that your codebase with be clean, as leaving unused code in your codebase make it unclean and difficult to read through in other to find and fix bug or understanding the code base

Reply:
- [ ] Please try to be polite while reviewing and remember to try to be more suggestive rather than commanding when giving a review. We don't want the receiver to feel like we are giving them orders so they feel open to accepting our feedback. Instead of starting with `Delete unused code ........` ,start with `I think it would be better to ........`
- [ ] You are good in explaining why request this change, in addition to I recommend to add link to verify with your changes.